tests/std/tests/P0220R1_string_view/test.cpp

```diff
@@ -303,6 +304,25 @@ constexpr bool test_case_buffer_constructor() {
     return true;
 }
 
+constexpr bool test_case_contiguous_constructor() {
+#ifdef __cpp_lib_ranges
+    const vector<char> expectedData{'n', 'o', ' ', 'n', 'u', 'l', 'l'};
+    // Also tests the corresponding deduction guide:
+    basic_string_view sv(expectedData.begin(), expectedData.end());
+    static_assert(is_same_v<decltype(sv), string_view>);

```

Comment:
Should this use Almost Always Concept style?
```suggestion
    same_as<string_view> auto sv = basic_string_view(expectedData.begin(), expectedData.end());
```